src/clang-c-frontend/clang_c_convert.cpp

```diff
@@ -1608,28 +1608,31 @@ bool clang_c_convertert::get_expr(const clang::Stmt &stmt, exprt &new_expr)
     const clang::MemberExpr &member =
       static_cast<const clang::MemberExpr &>(stmt);
 
+    exprt comp;
+    if(get_decl(*member.getMemberDecl(), comp))
```

Comment:
I think the if I mentioned should be placed inside the `if (!perform_virtual_dispatch)`, as comp is never used in the other side of that branch.

Reply:
Somehow I missed the email notification for this comment. I'll fix it in the next patch.